src/store/fileUploadStore.js

```diff
@@ -349,7 +350,12 @@ const actions = {
 				context.commit('markFileAsPendingUpload', { uploadId, index, sharePath: uniquePath })
 			} catch (exception) {
 				console.error(`Error while uploading file "${fileName}":` + exception.message, fileName)
-				showError(t('spreed', 'Error while uploading file "{fileName}"', { fileName }))
+				if (exception.response) {
+					const message = await parseUploadError(exception)
+					if (message) {
+						showError(message)
+					}
```

Comment:
Hmm should we:
```suggestion
					} else {
						showError(t('spreed', 'Error while uploading file "{fileName}"', { fileName }))
					}
```

Reply:
yep, would be as before for unhandled cases

Reply:
oops, I removed it by mistake